src/style.css

```
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  --bigIconSize: 19px;
+  --smallIconSize: 11px;
+}
+
+.search__icon,
+.search--big .search__icon,
+.search--small .search__icon {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  position: relative;
+}
+
+.search--big .search__icon {
+  background-size: cover;
+  width: 19px;
+  height: 19px;
+  min-width: var(--bigIconSize);
+  min-height: var(--smallIconSize);
+  top: 25px;
+  left: 26px;
+  margin-right: -19px;
+}
+
+.search--small .search__icon {
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  min-width: var(--smallIconSize);
+  min-height: var(--bigIconSize);
+  top: 15px;
+  left: 13px;
+  margin-right: -11px;
+}
+
+.search {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.search--big {
+  height: 70px;
+  margin-top: 20px;
+}
+
+.search--small {
+  height: 42px;
+  margin-top: 20px;
+}
+
+.search input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.2);
+  font-family: Avenir, sans-serif;
+}
+
+.search input::placeholder {
+  color: #3d4e61;
+}
+
+.search--big input {
```

```suggestion
.search--big .input {
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**